pkg/workload/workload.go

```diff
@@ -771,6 +771,22 @@ func IsActive(w *kueue.Workload) bool {
 	return ptr.Deref(w.Spec.Active, true)
 }
 
+func IsPodsReadyStartCond(w *kueue.Workload) *metav1.Condition {
+	cond := apimeta.FindStatusCondition(w.Status.Conditions, kueue.WorkloadPodsReady)
+	if cond == nil || cond.Reason != kueue.WorkloadWaitForPodsStart {
+		return nil
+	}
+	return cond
+}
```

Comment:
```suggestion
	if cond := apimeta.FindStatusCondition(w.Status.Conditions, kueue.WorkloadPodsReady); 
	   cond != nil && cond.Reason == kueue.WorkloadWaitForPodsStart {
		return cond
	}
	return nil
}
```
nit, to avoid negating the reason equality which makes it harder to imo.

Reply:
Done